### ISSUE TITLE
Usersnap Feedback - Would be nice if you could see the red pins for featured entries on the ma[...]

### ISSUE THREAD

**Issue by ascott** [participedia/usersnaps#901]
**Sender**: jesi.carson@gmail.com
**Comment**: Would be nice if you could see the red pins for featured entries on the map... they get buried below the other black pins. Is there a way to surface them?
[Open #595 in Usersnap Dashboard](https://usersnap.com/a/#/participedia-dev/p/participedia-net-9cb6d041/595)

<a href='https://usersnappublic.s3.amazonaws.com/2019-10-17/00-59/364fdd6f-c01f-414b-accb-511313f89946.png'>![Usersnap Feedback - Would be nice if you could see the red pins for featured entries on the ma(...)](https://usersnappublic.s3.amazonaws.com/2019-10-17/00-59/5e2b748c-371a-417a-819f-7e3bd3dcbc4c.jpg)</a>
<a href='https://usersnappublic.s3.amazonaws.com/2019-10-17/00-59/364fdd6f-c01f-414b-accb-511313f89946.png'>Download original image</a>

**Browser**: Chrome 77 (Macintosh)
**Referer**: [https://participedia.net/](https://participedia.net/)
**Screen size**: 1680 x 1050 **Browser size**: 1680 x 948

Powered by [usersnap.com](https://usersnap.com/?utm_source=github_entry&utm_medium=web&utm_campaign=product)

**Comment by ascott** [comment]
@jesicarson i think we should see if using the clustering on the pins will help to show off the featured pins more. i could also investigate how the order of the pins impacts how they are rendered on the map. It might be that the last pins in the list are rendered on top, in which case, i could reorder the list of pins before passing them to the map, but i will have to investigate further.

**Comment by jesicarson** [comment]
Yes let's add this to a future review of the map overall. So far, considerations include:
- seeing featured items (this issue)
- surfacing multiple entries with the same lat/long (discussed in Nov 4 D&T agenda)
- moving the map to its own (full) page, possibly accessed in a similar way as the list/grid views (suggested during user testing)
- replacing prime real estate with something that better helps users understand the site / get into the content (suggested during user testing)
